### DISCUSSION TITLE
Dir-structure for python-app.yml

### DISCUSSION THREAD

**Discussion by buchmann-dhi** [DHI/python-package-development#16] [General]
Hi @ecomodeller  and @jsmariegaard,
Just a question to get started on assignment 4.
In the assignment
 https://dhi.github.io/python-package-development/projects/data_cleaning/Project_module_04.html
 it is stated that "python-app.yml [is] in the .github folder"
 Actually, it seems that it is in .github/workflows
 This leads to the possible target location in our own repos, as it should "sit in the same folder".
 But is that .github/ or .github/workflows
 ?
 I'm just trying to iron out a possible unclear point, which could affect other participants as well.
 /Bjarne
 PS: For now I will assume that the location must be github/workflows

**Comment by ecomodeller** [comment]
It is `.github/workflows`

If in doubt, take a look at the officical docs, e.g [Quickstart for GitHub Actions](https://docs.github.com/en/actions/quickstart)
